packages/typespec-python/tasks.py

```diff
@@ -157,7 +157,7 @@ def all_specification_folders() -> List[Path]:
 def regenerate(c, name=None, debug=False):
     specs = [
         s for s in all_specification_folders()
-        if any(f for f in s.iterdir() if f.name == "main.tsp") and "type/model/inheritance/enum-discriminator" not in s.as_posix()
+        if any(f for f in s.iterdir() if f.name == "main.tsp")
```

Comment:
why was this code originally here? In the future if we want to skip tests we can create a constant with the list of tests to skip so that way it's more readable

Reply:
Make sense and will do it in https://github.com/Azure/autorest.python/pull/2108/files#r1324125711